internal/controller/csiaddons/encryptionkeyrotationjob_controller.go

```diff
@@ -0,0 +1,332 @@
+/*
+Copyright 2022 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	corev1 "k8s.io/api/core/v1"
+	scv1 "k8s.io/api/storage/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	csiaddonsv1alpha1 "github.com/csi-addons/kubernetes-csi-addons/api/csiaddons/v1alpha1"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/connection"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/proto"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/util"
+	"github.com/csi-addons/spec/lib/go/identity"
+	"github.com/go-logr/logr"
+)
+
+// EncryptionKeyRotationJobReconciler reconciles a EncryptionKeyRotationJob object
+type EncryptionKeyRotationJobReconciler struct {
+	client.Client
+	Scheme   *runtime.Scheme
+	ConnPool *connection.ConnectionPool
+	Timeout  time.Duration
+}
+
+// +kubebuilder:rbac:groups=csiaddons.openshift.io,resources=encryptionkeyrotationjobs,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=csiaddons.openshift.io,resources=encryptionkeyrotationjobs/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=csiaddons.openshift.io,resources=encryptionkeyrotationjobs/finalizers,verbs=update
```

Comment:
remove extra space

Reply:
Removed. I would like to add that the newest version of kubebuilder now adds a space by default.